packages/react/src/accordion/Accordion.tsx

```diff
@@ -0,0 +1,101 @@
+import { Heading } from '../content';
+
+type AccordionProps = {
+  children: React.JSX.Element[] | React.JSX.Element;
+
+  /** Additional CSS className for the element. */
+  className?: string;
+
+  /** Additional style properties for the element. */
+  style?: React.CSSProperties;
+};
+type AccordionItemProps = {
+  children: React.JSX.Element[];
+
+  /** Additional CSS className for the element. */
+  className?: string;
+
+  /** Additional style properties for the element. */
+  style?: React.CSSProperties;
+};
+
+function Accordion(props: AccordionProps, ref: Ref<HTMLDivElement>) {
+  return (
+    <div
+      className={cx('flex flex-col rounded-lg bg-white p-2', props.className)}
+      ref={ref}
+    >
+      {props.children}
+    </div>
+  );
+}
+
+function AccordionItem(props: AccordionItemProps, ref: Ref<HTMLDivElement>) {
+  const [open, setOpen] = useState(false);
+  const contentId = useId();
+
+  return (
+    <div
+      className={cx(
+        'group relative px-2',
+        /** Pseudoelement for the gray bottom border */
+        'after:absolute after:left-[9px] after:right-[9px] after:h-px after:bg-gray-light after:last:h-0',
+      )}
+      data-open={open || undefined}
+      ref={ref}
+    >
+      {Children.map(props.children, (child) => {
+        /** Since we are using <Heading> and <Content> from Grunnmuren to
+         * render the accordion item, we can use the level-prop to identify the heading component */
+        if (child.props.level) {
+          return (
+            <Heading level={child.props.level}>
+              <button
+                onClick={() => setOpen(!open)}
+                className={cx(
+                  'flex w-full flex-1 items-center justify-between rounded-md py-[18px] text-left font-semibold leading-7',
+                )}
+                aria-expanded={open}
+                aria-controls={contentId}
+              >
+                {child.props.children}
+                <ChevronDown
+                  className={cx(
+                    'text-base transition-transform duration-150 group-data-[open]:rotate-180 motion-reduce:transition-none',
+                  )}
+                />
+              </button>
+            </Heading>
+          );
+        } else {
+          return (
+            <div
+              aria-labelledby={contentId}
+              role="region"
+              className={cx(
+                '[&_[data-slot="content"]]:mb-[10px] [&_[data-slot="content"]]:border-l-[3px] [&_[data-slot="content"]]:border-mint [&_[data-slot="content"]]:px-3.5 [&_[data-slot="content"]]:py-1.5',
+                'grid transition-all duration-300',
+                open ? 'grid-rows-[1fr]' : 'grid-rows-[0fr]',
+              )}
+              // @ts-expect-error type error until this is added to react: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60822
+              inert={open || undefined}
+            >
+              <div className="overflow-hidden">{child}</div>
```

Comment:
Nå har vi endt opp med 2 wrapper divs rundt `Content`. Jeg føler at APIet vårt begynner å bryte sammen her.

Hvorfor trenger vi disse to wrapperne, er det mulig å få plassert alt dette rett på Content-komponenten i stedet for?

Reply:
Ja, det er vel mulig å slenge på `overflow-hidden` på child her og fjerne den ene diven på den måten. Hvis vi ønsker å putte alt inn på content må vi ha en annen animasjon av høyden.

Reply:
Checka den sista kommentaren jag hade på PRen: När jag försöker med det så landar det i att antingen den left bordern syns oavsett, eller att layouten hoppar lite. Om ni har inspill här så är det digg, för jag känner att jag inte helt får till det atm

Reply:
Kan vi flytte griden til feks accordion item? Grid med en heading area og en content area. Funker animasjonen fortsatt da?

Reply:
Nei, dessverre. Må var en grid med kun én child og én row

Reply:
Ja, animationen fungerar på den måten, men när jag gjorde det sån blev det antingen problem med pseudoelementet eller den bordern som vi placerar med [&_[data-slot="content"]]

Reply:
Se den bilden jag limmade in ovanför dina kommentarer, kommer liksom dit, men får det inte att bli pent

Reply:
Så beste jeg kan komme på for å holde animasjonen enkel (bruke grid) er:

```
<div
  id={contentId}
  aria-labelledby={buttonId}
  role="region"
  className={cx(
    '[&_[data-slot="content"]]:mb-[10px] [&_[data-slot="content"]]:border-l-[3px] [&_[data-slot="content"]]:border-mint [&_[data-slot="content"]]:px-3.5 [&_[data-slot="content"]]:py-1.5',
    'grid transition-all duration-300 [&_[data-slot="content"]]:overflow-hidden',
    open ? 'grid-rows-[1fr]' : 'grid-rows-[0fr]',
  )}
  inert={open ? undefined : 'true'}
>
  {child}
</div>
```

Men da må vu flytte den mint-borderen

Reply:
> Så beste jeg kan komme på for å holde animasjonen enkel (bruke grid) er:
> 
> ```
> <div
>   id={contentId}
>   aria-labelledby={buttonId}
>   role="region"
>   className={cx(
>     '[&_[data-slot="content"]]:mb-[10px] [&_[data-slot="content"]]:border-l-[3px] [&_[data-slot="content"]]:border-mint [&_[data-slot="content"]]:px-3.5 [&_[data-slot="content"]]:py-1.5',
>     'grid transition-all duration-300 [&_[data-slot="content"]]:overflow-hidden',
>     open ? 'grid-rows-[1fr]' : 'grid-rows-[0fr]',
>   )}
>   inert={open ? undefined : 'true'}
> >
>   {child}
> </div>
> ```
> 
> Men da må vu flytte den mint-borderen

Kunne disse propsene/attrbiuttene også vært rett på content-diven, i stedet for wrappet rundt?

Reply:
Tenker du på `id`, `role`, `aria-label`, `inert` osv? De kan det, men da må vi vel bruke `React.cloneElement` for å sende de videre til `child`? Blir ikke det litt heavy?

Reply:
Ikke nødvendigvis. Vi kan gjenbruke RAC primitivene med slots og context

Reply:
Ah smart, ja det kan vi jo gjøre! Tenker du da at vi lager custom `ContentContext` som har alle disse HTML-atributtene som optional? Eller er det en ferdig Context fra RAC vi kan bruke?